spec/controllers/users_controller_spec.rb

```diff
@@ -47,6 +47,19 @@
       expect(FinePrint.signed_contract?(temp_user, contract_1)).to eq true
       expect(FinePrint.signed_contract?(temp_user, contract_2)).to eq true
     end
+
+    it "claims an unclaimed account" do
+      user = FactoryGirl.create :user, state: 'unclaimed'
+      expect(user.state).to eq 'unclaimed'
+      controller.sign_in! user
```

Comment:
The account is claimed whenever a user confirms a matching email. At that point it's merged with their existing account